src/v2/controllers/FormController.js

```diff
@@ -156,6 +156,19 @@ export default Controller.extend({
         .then(this.handleIdxSuccess.bind(this))
         .catch(error => {
           this.showFormErrors(this.formView.model, error);
+        })
```

Comment:
How about emit an event and let the `BaseLoginRouter` listen on it? in this way, we can centralize the reset/init logic in one place.

Reply:
We should not initiate login flow from FormController..

Reply:
@shuowu @pradeepdewda-okta Please review changes

Reply:
I think instead, you should trigger BaseLoginRouter's `render` function instead that starts the loginflow and handle cleanup..

Reply:
Done